docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -426,6 +426,49 @@ incorporate the processes associated with crime reporting, the offense-trial
 pipeline, and an awareness of how the relationship between various social actors and
 the algorithm may impact the intended outcomes of a given model.
 
+
+The 4/5ths Rule: A Common Trap
+------------------------------
+
+In the literature around fairness in machine learning, one will often find
+the so-called "4/5ths rule" or "80% rule" used to assess whether a model
+(or mitigation technique) has produced a 'fair' result.
+This rule is an implementation of the demographic parity ratio introduced
+in :ref:`disparity_metrics` section, with a result considered 'fair' if the ratio
+exceeds 80% for all identified subgroups.
+*Unthinking application of this threshold is an extremely common abstraction trap.*
+
+The trap springs on three levels:
+
+1. Where is it valid?
+2. What does it mean, legally?
+3. Why 80%?
+
+*Where is the 4/5ths rule valid?*
+The answer is that it is only valid
+within a specific area of US federal employment law.
+Outside this context, the rule has no particular claim to validity.
+
+*What exactly does the rule mean?*
+Legally, the 80% threshold is a limit for
+`prima facie evidence <https://en.wikipedia.org/wiki/Prima_facie>`_
+that illegal discrimination has occurred.
+However, is does not prove or disprove any case.
+It is possible to argue successfully that a demographic parity
+ratio of 50% is still fair.
+It is equally possible to argue that even at a 95% ratio, illegal
+discrimination has occurred.
+
+*Why is the ratio set at 80%?*
+So far as we can determine, the value of 80% was negotiated
+as a political compromise.
```

Comment:
I would love to hear more about this. Sounds like a terribly interesting article that I will be happy to read sometime 🙂

Reply:
"Political compromise" doesn't seem like the right terminology here. If I understand correctly what Jiahao said, federal lawmakers looked at two different state-level legislations for references, one with threshold 0.9 and the other with threshold 0.7, and decided to use 0.8 since it's the average of the two thresholds.  

There really wasn't a compromise in the sense that two parties came to an agreement to this value after some debate.

Reply:
My impression was that the original debate had been just within California (where the basic idea originated), and the two groups there had been advocating for 0.7 and 0.9.... and that the 0.8 was a compromise between the two. This is why we need the actual citations :-)